api/src/services/application-exporter.service.ts

```diff
@@ -32,7 +32,7 @@ view.csv = {
   },
   listings: false,
 };
-const NUMBER_TO_PAGINATE_BY = 250;
+const NUMBER_TO_PAGINATE_BY = 500;
```

Comment:
Let's keep this as a hard coded value, but if we have to do some tinkering it could be useful to have this an env variable.